File: Entities/Concrete/Stock.cs
using Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Concrete
{
    public class Stock : BaseEntity, IEntity
    {
        public int ProductId { get; set; }
        public Product Product { get; set; }
        public int Quantity { get; set; }
        public bool IsReadyForSale { get; set; }
    }
}

File: UIPreparation/UI/src/app/core/components/app/stock/model/Stock.ts
export class Stock {
  id: number;
  productName: string;
  productId: number;
  quantity: number;
  isReadyForSale: boolean;
  isDeleted: boolean;
  colorId: number;
  colorName: string;
  colorCode: string;
}

export class Product {
  id: number;
  name: string;
}
